styles/scss/_page.scss

```diff
@@ -6,3 +6,9 @@
         border-top: 0;
     }
 }
+
+.page-sub-section {
+    border-bottom: 1px solid #DDD;
+    padding-top: 20px;
+    padding-bottom: 20px;
```

Comment:
just a tip: you can group vertical or horizontal paddings with the same value like: padding: 20px 0;